### PR TITLE
docs: fix color contrast to improve DX

### LINKED ISSUE #2187: Docs: Improve contrast in dark mode
Following components documentation needs to be updated with better contrast in dark mode to improve the DX.

- eek
- image-placeholder
- Labelling menu button

### PR DESCRIPTION
**Pull request by saiponnada**
Fixes #2187

- [x] This PR contains CSS changes
- [ ] This PR does not contain CSS changes

## Description
Made few CSS changes to improve contrast while reviewing documentation in dark mode.

## Notes
Image placeholder's `svg` might not be visible in dark mode. Checkout the screenshot below. This placeholder svg might need to be replaced in future.

## Screenshots
**Image Placeholder**
Before
image

After
image

**EEK**
Before
image

After
image

**Menu Button highlight**

Before
image

After
image

## Checklist

- [x] I verify the build is in a non-broken state
- [X] I verify all changes are within scope of the linked issue

- [x] I regenerated all CSS files under dist folder
- [x] I tested the UI in all supported browsers
- [ ] I did a visual regression check of the components impacted by doing a Percy build and approved the build
- [x] I tested the UI in dark mode and RTL mode
- [ ] I added/updated/removed Storybook coverage as appropriate

### CODE REVIEW

File: docs/_includes/image-placeholder.html [hunk 1]
```diff
@@ -28,15 +28,15 @@ <h3>Variable widths with borders</h3>
...
+                    style="border: 1px solid orange; height: 150px; width: 150px"
                     role="img">
                     {% include symbol.html name="placeholder" prefix="image" %}
                 </svg>
                 </svg>
             </div>
             <div>
                 <svg aria-label="image not found" class="image-placeholder" focusable="false"
-                    style="border: 1px dotted black; height: 250px; width: 250px" role="img">
+                    style="border: 1px dotted orange; height: 250px; width: 250px" role="img">
```
Comment (ArtBlue) [inline docs/_includes/image-placeholder.html:39]: I don't know how far we want to take this, but it looks like this change appears to create a color contrast issue on light mode. The dotted orange seems too faint. 
image

Comment (saiponnada) [reply docs/_includes/image-placeholder.html:39]: We might need to pick a color that improves the contrast then. Can make that change.

**Review by ArtBlue — COMMENTED** [review]
If this effort is to fix DX for dark mode, we should use `css` props for the blue border for the div above too. 
image

Also, for some reason, the Image Placeholder link in the sidebar doesn't take me to the Image Placeholder section though the link and div id seem to match. Weird.

**Comment by saiponnada** [comment]
> Also, for some reason, the Image Placeholder link in the sidebar doesn't take me to the Image Placeholder section though the link and div id seem to match. Weird.

I noticed that. My assumption is it's because the `svg` also has the same id 
`<symbol viewBox="0 0 300 300" id="image-placeholder">...</symbol>`

**Review by ArtBlue — APPROVED** [review]
Looks good.

**Pull request merged**